api/src/hooks/locokit/index.ts

```diff
@@ -23,14 +23,18 @@ export async function setLocoKitContext(context: HookContext) {
   locokitContextLogger.info('starting on service "%s" ...', context.service.constructor.name)
   const { transaction } = context.params
 
-  context.$locokit = {}
+  context.params.$locokit = {}
 
-  const { workspaceSlug, datasourceSlug /* tableSlug, datasetSlug */ } = context.params?.route || {}
+  const { workspaceSlug, datasourceSlug, tableSlug, datasetSlug, workflowSlug } =
+    context.params?.route || {}
```

Comment:
```suggestion
    context.params.route ?? {}
```
Does the linter pass if you make this modification?

Reply:
I update to `context.params?.route ?? {}` as `params` can be `undefined`.